# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/prim_store/interned.rs
# Commit: ea22ccce0c4d
# Full Hash: ea22ccce0c4dbe8a1c7ace807924275497f6c899
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-05-25 09:36:24
# Regressor Bug: 1769963
# File Overlap Count: 1
# Description:
#   Bug 1769963 - Fix backdrop-filter inside complex transform with CSS filter chain r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D147124
# ==============================================================================

diff -r 60c64f026b6b -r ea22ccce0c4d gfx/wr/webrender/src/prim_store/interned.rs
--- a/gfx/wr/webrender/src/prim_store/interned.rs	Wed May 25 05:48:08 2022 +0300
+++ b/gfx/wr/webrender/src/prim_store/interned.rs	Wed May 25 02:59:32 2022 +0000
@@ -4,7 +4,7 @@
 
 // list of all interned primitives to match enumerate_interners!
 
-pub use crate::prim_store::backdrop::Backdrop;
+pub use crate::prim_store::backdrop::{BackdropCapture, BackdropRender};
 pub use crate::prim_store::borders::{ImageBorder, NormalBorderPrim};
 pub use crate::prim_store::image::{Image, YuvImage};
 pub use crate::prim_store::line_dec::{LineDecoration};